Azure/sonic-mgmt
# [bgpmon]Add new test case test_bgpmon

Signed-off-by: bingwang <bingwang@microsoft.com>


### Description of PR


Summary:
Fixes # (issue)
This PR implements a new test case ```test_bgpmon``` according following test plan.

**Testplan:**
1. Choose some ip address which could be reached from the DUT through default routing only
2. Configure BGPMON neighbor peer with the IP address from the previous item on DUT
3. Start capturing traffic on DUT interfaces, which are used for the default routes
4. The test is successfull if we found a syn tcp packet to the ip, and dst port 179 on one of the interfaces. We should capture the traffic for 2 minutes.
By default bgp connection timer is 120 seconds

### Type of change




### Approach
#### What is the motivation for this PR?
This PR is to add a new test case for bgp monitor.

#### How did you do it?

#### How did you verify/test it?
Verified on Arista-7260cx3 (t0) and vtestbed (t1)
```
py.test --inventory ../ansible/str,../ansible/veos --host-pattern str-7260cx3-acs-2 --module-path ../ansible --testbed vms7-t0-7260-2 --testbed_file ../ansible/testbed.csv --junit-xml=tr.xml --log-cli-level warn --collect_techsupport=False --topology=t0,any,util bgp/test_bgpmon.py
========================================================================================= test session starts =========================================================================================
collected 1 item                                                                                                                                                                                      

bgp/test_bgpmon.py::test_bgpmon PASSED                                                                                                                                                          [100%]

------------------------------------------------------------------ generated xml file: /data/Networking-acs-sonic-mgmt/tests/tr.xml -------------------------------------------------------------------
====================================================================================== 1 passed in 34.23 seconds ======================================================================================
```
```
py.test --inventory ../ansible/veos_vtb --host-pattern vlab-01 --module-path ../ansible/library/ --testbed vms-kvm-t1 --testbed_file ../ansible/vtestbed.csv --junit-xml=tr.xml --log-cli-level warn -k 'not test_restart_syncd' --topology=t1,t0,any,util bgp/test_bgpmon.py
========================================================================================= test session starts =========================================================================================
collected 1 item                                                                                                                                                                                      

bgp/test_bgpmon.py::test_bgpmon PASSED                                                                                                                                                          [100%]

-------------------------------------------------------------------------- generated xml file: /data/sonic-mgmt/tests/tr.xml --------------------------------------------------------------------------
====================================================================================== 1 passed in 34.37 seconds ======================================================================================
```
#### Any platform specific information?
No.

#### Supported testbed topology if it's a new test case?
Support any topo.

### Documentation
